src/main/java/com/esri/samples/tiledlayers/export_vector_tiles/ExportVectorTilesSample.java

```diff
@@ -0,0 +1,228 @@
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.tiledlayers.export_vector_tiles;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.concurrent.ExecutionException;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.geometry.Point2D;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Button;
+import javafx.scene.control.ProgressBar;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.concurrent.Job;
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.data.VectorTileCache;
+import com.esri.arcgisruntime.geometry.Envelope;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.layers.ArcGISVectorTiledLayer;
+import com.esri.arcgisruntime.layers.Layer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.ItemResourceCache;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.portal.PortalItem;
+import com.esri.arcgisruntime.security.Credential;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+import com.esri.arcgisruntime.tasks.vectortilecache.ExportVectorTilesJob;
+import com.esri.arcgisruntime.tasks.vectortilecache.ExportVectorTilesParameters;
+import com.esri.arcgisruntime.tasks.vectortilecache.ExportVectorTilesResult;
+import com.esri.arcgisruntime.tasks.vectortilecache.ExportVectorTilesTask;
+
+public class ExportVectorTilesSample extends Application {
+
+  private MapView mapView;
+
+  @Override
+  public void start(Stage stage) throws Exception {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Export Vector Tiles Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // set the map to the map view
+      mapView = new MapView();
+
+      // authenticate with an organization account on arcgis.com
+      AuthenticationDialog authenticationDialog = new AuthenticationDialog();
+      authenticationDialog.setOnCloseRequest(r -> {
```

Comment:
I believe it would be better to just use the default authentication dialog, `AuthenticationManager.setAuthenticationChallengeHandler(new DefaultAuthenticationChallengeHandler());` . Then you will not need to create your own.

Look something like,
```java
// set the map to the map view
      mapView = new MapView();

      // authenticate with an organization account on arcgis.com
      AuthenticationManager.setAuthenticationChallengeHandler(new DefaultAuthenticationChallengeHandler());

      Portal portal = new Portal("http://www.arcgis.com", true);
      PortalItem portalItem = new PortalItem(portal, "86f556a2d1fd468181855a35e344567f");
      portalItem.addDoneLoadingListener(() -> {
        if (portalItem.getLoadStatus() == LoadStatus.LOADED) {
          // loading the vector tiled layer will invoke the authentication challenge
          ArcGISVectorTiledLayer vectorTiledLayer = new ArcGISVectorTiledLayer(portalItem);
          ArcGISMap map = new ArcGISMap(new Basemap(vectorTiledLayer));
          mapView.setMap(map);
        } else {
          Alert alert = new Alert(Alert.AlertType.ERROR, "Portal Item: " + portalItem.getLoadError().getCause().getMessage());
          alert.show();
        }
      });
      portalItem.loadAsync();

      // create a graphics overlay for the map view
      GraphicsOverlay graphicsOverlay = new GraphicsOverlay();
      mapView.getGraphicsOverlays().add(graphicsOverlay);

      // .... rest of code
```